web/urls.py

```diff
@@ -17,4 +17,5 @@
 	url(r'^privacy/$', views.privacy, name='pages-privacy'),
 	url(r'^project/(?P<project_id>[0-9]+)/$', views.project_detail),
 	url(r'^logout/$', views.logout, name='logout'),
+	url(r'^member_page/$', views.member_page, name='pages-member_page'),
```

Comment:
And also the notation is not very nice. Why is `member_page` written with underscore and then in the name attribute you have `pages-member_page` make them all use dashes instead of underscores.